src/mmrelay/plugin_loader.py

```
@@ -870,485 +897,750 @@ def _check_auto_install_enabled(config):
+                    cwd=plugins_dir,
+                    timeout=120,
+                )
+                if ref_type == "branch":
+                    logger.info(
+                        f"Cloned repository {repo_name} from {repo_url} at branch {ref_value}"
+                    )
+                else:
+                    logger.info(
+                        f"Cloned repository {repo_name} from {repo_url} at tag {ref_value}"
+                    )
+                return True
```

Ruff (0.14.3)

Consider moving this statement to an `else` block

(TRY300)